dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -409,7 +422,8 @@ def _test_logged_in_milestone_should_save_gallery_when_passing_an_impressive_lev
 
     assert_response :success
 
-    expected_response = build_expected_response(level_source: "http://test.host/c/#{assigns(:level_source).id}")
+    obfuscated_level_source_url = assigns(:level_source).obfuscate_level_source_id(current_user_id)
+    expected_response = build_expected_response(level_source: "http://test.host/r/#{obfuscated_level_source_url}")
```

Comment:
It would. That said, I'm calling that part of the "rewritten to be readable" in the PR summary, that I'm hoping to punt on as part of this PR. Expect a follow-up...